spread.yaml

```diff
@@ -82,7 +82,7 @@ backends:
         FATAL "$SPREAD_SYSTEM is not supported!"
       fi
 
-      multipass launch --disk 20G --memory 2G --name "$instance_name" "$image"
+      multipass launch --cpus 8 --disk 30G --memory 8G --name "$instance_name" "$image"
```

Comment:
This change can easily cause a computer to run out of memory. In this repository, spread tests run in parallel for Focal, Jammy, and Noble, so it will be 3x8GB = 24GB.

Additionally, there is no guarantee that most computers will have 8 cores available—1 CPU is always guaranteed. Therefore, these changes are suitable only when applied locally on a machine that is guaranteed to have those resources.